spec.yaml

```diff
@@ -143,7 +187,29 @@ paths:
                 "general": {
                   "installation_name": "shire",
                   "provider": "aws",
-                  "datacenter": "eu-central-1"
+                  "datacenter": "eu-central-1",
+                  "availability_zones": {
+                    "max": 3,
+                    "default": 1
+                  }
```

Comment:
Can we add the actual list of values as well? So far I thought we use the info endpoint internally as well which would make the work with it easier. Otherwise there has to be a separate mechanism to get them anyway.

Reply:
what do you mean? the names of the availability zones? why do we need them?

Reply:
We need them latest in the `aws-operator`. So where do we get them from then? My thinking was that we simply use the information from the info endpoint internally as well and it would also be more informational for the customer when using the endpoint.

Reply:
I read the randomization spec and I like it. We can do this. This is separate from having the real AZ names in the info endpoint though. Why should we not expose them here? Having information in the info endpoint is pretty good IMO.

Reply:
I see no reason why we should expose them.

Reply:
I gave you a reason already. We can use the information provided by the info endpoint internally because we need these information. We can use the info endpoint on our own as interface. I would like to have the AZ names in here for that purpose. The alternative is to have some other mechanism which is then some other interface and we could just make it easier and dog food that way.

Reply:
> We can use the information provided by the info endpoint internally because we need these information

Internal reason don't justify exposing this externally through the api. and i don't see why something like cluster-service should suddenly connect via the api. let keep things minimal. It is not needed in the interface and thats why it won't be exposed. 

the other problem i have with this is that we expose information that would lead to false assumptions. if you specify one availability zone it won't end up in the first AZ. of course we can describe this. maybe we should provide more info about this.

Reply:
Disagree and commit.